src/lib/warn-upload-overwrite.js

```diff
@@ -22,6 +24,25 @@ const getEnvironmentKey = () => {
   return `${parsed.hostname}${path}`;
 };
 
+const getCompressibleTypes = async () => {
+  const parsedUrl = url.parse(environment.apiUrl);
+  const configUrl = `${parsedUrl.protocol}//${parsedUrl.auth}@${parsedUrl.host}/api/couch-config-attachments`;
+  try {
+    if (cache.has('compressibleTypes')) {
+      return cache.get('compressibleTypes');
+    }
+    const resp = await request.get({ url: configUrl, json: true });
+    cache.set('compressibleTypes', resp.compressible_types);
+    return resp.compressible_types;
+  } catch(e) {
+    cache.set('compressibleTypes', null);
+    e.statusCode === 404 ?
+      log.info('couch-config-attachments endpoint not found') :
+      log.info('Error trying to get config', e);
```

Comment:
this should be an error.  Also, please have a look at how `medic-conf` log module is used: https://github.com/medic/medic-conf/blob/71642859987fe98a8d6d6d1d3dcccbe8b8a0722c/src/lib/upload-forms.js#L6